.github/workflows/extract-issue-links.yml

```diff
@@ -103,3 +112,27 @@ jobs:
             }
         env:
           SLACK_WEBHOOK_URL: ${{ steps.secrets.outputs.sec-webhook }} # from previous step that fetches secrets from Vault
+      - name: Determine necessity to send translation notification
+        run: |
+          if [[ "${{ inputs.versions }}" =~ (,|^)[[:space]]*${{ inputs.translation-notification-matcher }}[[:space]]*(,|$) ]]; then   
+            echo "NOTIFY_TRANSLATIONS=true" >> $GITHUB_ENV;
```

Comment:
Just a hint for future workflow projects. I don't see the need to act on this: 
If only passing data inside the workflow and no need to make the data visible on the runner executing the workflow  (like environment variables do), it's also possible to define outputs: https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs

Reply:
Thanks for sharing, I wasn't aware of `$GITHUB_OUTPUT` yet.